### DISCUSSION TITLE
Version 4.0.98.1-alpha `<Alpha 1>` edition

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MIR#39] [Announcements]
### Version 4.0.98.1-alpha `<Alpha 1>` edition

* 18 Sep 2022
* For Kerbal Space Program 1.12.3

##### Rename 4.0.98.1

* Internal Part names should not change going forward
* <docs/PartsCatalog.md> v1.1.4.0 now shows all parts
* All parts have received initial
  * update pass
  * localization pass
  * asset pass
* <MIR.version>
  * removed [KSP_VERSION_MAX]
  * added SubAssemblies
* Files
  * <TKS_Zarya.cfg> --> <mir-tks-zarya.cfg> v1.2.0.0
  * <TKS_Kristall.cfg> --> <mir-tks-kristall.cfg> v1.2.0.0
  * <TKS_Kvant2.cfg> --> <mir-tks-kvant-2v.cfg> v1.2.0.0
  * <TKS_Priroda.cfg> --> <mir-tks-priroda.cfg> v1.2.0.0
  * <TKS_Spektr.cfg> --> <mir-tks-spektr.cfg> v1.2.0.0
  * <TKS_tug.cfg --> <mir-tks-tug.cfg> v1.2.0.0
  * <MirCap.cfg> --> <mir-tks-dock-cap.cfg> v1.2.0.0
  * <MirDrogue.cfg> --> <mir-tks-dock-drogue.cfg> v1.2.0.0
  * <MirProbe.cfg> --> <mir-tks-dock-probe.cfg> v1.2.0.0
* Parts
  * TKS_Zarya --> mir-tks-zarya
  * TKS_Kristall -->  mir-tks-kristall
  * TKS_Kvant2V --> mir-tks-kvant-2v
  * TKS_Priroda --> mir-tks-priroda
  * MIR_Spektr1 --> mir-tks-spektr
  * TKS_tug1 --> mir-tks-tug
  * Mir_DockingPortProbe1 --> mir-tks-dock-probe
  * Mir_DockingPortCoverCap1 --> mir-tks-dock-cap
  * Mir_DockingPortDrogue1 --> mir-tks-dock-drogue
* Models
  * <model.mu> --> <mir-core.mu>
  * <MIR_core.mu> --> <mir-core.mu>
  * <model.mu> --> <mir-rcs-block-1.mu>
  * <NewModel.mu> --> <mir-solar.mu.mu>
  * <MIR_solartop.mu> --> <mir-solar-top.mu>
  * <Kvant_core.mu> --> <mir-kvant.mu>
  * <Kvant_solar.mu> --> <mir-kvant-solar.mu>
* Textures
  * <model000.dds> --> <MIR-DP00.dds>
  * <model001.png> --> <MIR-DP01.png>
* IVA
  * Filename
    * <Internal.cfg> --> <MIR-TKS-internal.cfg>
  * actually deactivate Spaces/MIR-core-internal/PropConfig.cfg
  Name
    * MIR_TKS_internal --> MIR-TKS-internal
    * update pointers in six parts

##### Split Out 4.0.98.1

* Update
  * <ECLSS.cfg>.cfg> v1.1.0.0

---

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/MIR/releases/tag/4.0.98.1-alpha'>Version 4.0.98.1-alpha `<Alpha 1>` edition</a>.</em>

**Comment by zer0Kerbal** [comment]
@Toa-Sampa 

* All parts
  * are now present; no further (internal) part name changes _should_ be required.
  * have had initial _localization_ pass completed. 
  * now are showing in editor
  * [PartsCatalog.md](https://zer0kerbal.github.io/MIR) now showing all parts; just needs better labels
  * Assets have been (as required) renamed, updated, and relocated.

ready now for you to begin the details - like what is missing.

I will go through the models and see if there are items not included in the part.cfg's (like lights).

**Reply by Toa-Sampa** [reply]
going to go ahead and install and start testing.  something I noticed in the previous alpha is that the MIR RCS seemed to be having texture issues (texture not showing up).  going to see if that is still the case.

**Reply by zer0Kerbal** [reply]
> going to go ahead and install and start testing. something I noticed in the previous alpha is that the MIR RCS seemed to be having texture issues (texture not showing up). going to see if that is still the case.

good eye - shouldn't be the case any longer: the rcs texture issue is resolved, had renamed the textures and either the model pointer changes didn't stick or I missed it or copied over them.

**Reply by Toa-Sampa** [reply]
alright.  Mir is now installing into my KSP, will power it up and check it out.  If I find a need to adjust the nodes, I will send whatever changes needed over to you.

**Reply by Toa-Sampa** [reply]
and that is going to be in testing in the stock environment, for now

**Reply by zer0Kerbal** [reply]
thank you.
can just zip up the .cfg's and drag'n'drop here.

**Reply by Toa-Sampa** [reply]
ok, the Mir RCS textures ARE back, but it could still use some work.  looks like the effects are completely whacked!

**Reply by zer0Kerbal** [reply]
good. and it is using the same effects from original release. I will pull the effects from another stock part which should also allow updating to RCSFX part module.

**Comment by Toa-Sampa** [comment]
Sounds good.

**Comment by Toa-Sampa** [comment]
Got it.
